src/server/services/tips-and-notifications.ts

```diff
@@ -92,12 +99,12 @@ export function getTipsAndNotificationsFromApiResults(
       continue;
     }
     // Collection of notifications
-    if ('notifications' in content) {
+    if ('notifications' in content && Array.isArray(content.notifications)) {
       notifications.push(...content.notifications);
     }
 
     // Collection of tips
-    if ('tips' in content) {
+    if ('tips' in content && Array.isArray(content.tips)) {
       for (const tip of content.tips) {
         // Should we show the tip as Notification?
```

Comment:
Is deze comment al beantwoordt? Anders even nog bespreken (met team waarschijnlijk) ? Ik wil ook wel zien hoe dit eruit gaat zien

Reply:
Dit is een nutteloze comment, de code zegt nl al `isTip`